docs/theme.js

```diff
@@ -107,11 +107,37 @@ export default {
       color: 'white',
     },
   },
+
+  pill: {
+    primary: {
+      container: {
+        py: 1,
+        borderRadius: 3,
+        bg: 'white',
+      },
+      content: {
+        color: 'primary',
+      },
+      icon: {},
+    },
+  },
+
+  images: {
+    close: {
+      height: '20px',
+      width: '20px',
+    },
+    pillIcon: {
+      width: '15px',
+      height: '15px',
+    },
```

Comment:
I dont think, we need styles for `pillIcon`. pillIcon must be a component prop for "Pill"

Reply:
Also, I don't see a pill icon prop for `Pill` component.